src/PrestaShopBundle/Install/XmlLoader.php

```diff
@@ -367,6 +368,58 @@ protected function getFallBackToDefaultEntityLanguage($iso, $entity)
         return file_exists($this->lang_path . $this->getFallBackToDefaultLanguage($iso) . '/data/' . $entity . '.xml') ? $iso : 'en';
     }
 
+    /**
+     * Special case for "country" entity.
+     */
+    public function populateEntityCountry()
+    {
+        $xml = $this->loadEntity('country');
+
+        // Read list of fields
+        if (empty($xml->fields)) {
+            throw new PrestashopInstallerException('List of fields not found for entity country');
+        }
+        $langs = [];
+        $languageList = LanguageList::getInstance();
+        foreach ($this->languages as $id_lang => $iso) {
+            $languageList->setLanguage($iso);
```

Comment:
I see what you're doing with the static variables reset and all But wouldn't it be cleaner and more stable to add a `getCountriesByLanguage` method instead in `LanguageList` ?